cmd.md

```diff
@@ -96,34 +103,59 @@ OR
 ## Creating files
 
 - Use `type nul > file_name.ext` to create an empty file in the cwd.
-- Use `echo enter text here > file_name.ext` to create a file with data in it (data without spaces is accepted).
+- Use `echo "enter text here" > file_name.ext` to create a file with data in it.
+
+OR
+
+- Use `copy con file_name.ext` to create a file. It will wait for input. Enter the input and hit <kbd>Ctrl</kbd> + <kbd>Z</kbd> and then the <kbd>Enter</kbd> key when done with the input.
 
 OR
 
-- Use `copy con file_name.ext` to create a file. It will wait for input. Enter the input (`enter` key can be used to go to the next line) and hit `ctrl + z` and then the `enter` key when done with the input.
+- Use `echo. > file_name.ext` to create an empty file. Note that there is no space between `echo` and `.`.
 
-## Writing to a file
+## The `find` command
 
-- If the same file is written to using a command like `echo enter text here > file_name.ext`, the original contents are overwritten.
-- To append to a file, use `echo enter text to be appended here >> existing_file_name.ext` and the text will be appended on the next line of the file.
-- To write a command's output to a file, use `command > file_name.ext`
+- The `find` command will look for a word/phrase in a file or folder.  
+  Eg: `find "Hello" hello.txt` will print all the lines with the word `Hello` in the file `hello.txt`.
+- By default, the text is case sensitive (`Hello` is different from `hello`). Use the `/i` flag to ignore case.  
+  Eg: `find /i "Hello" hello.txt`.
+- You can also tell it to search all the files in the current folder with `find "Hello" *`, or another folder with `find "Hello" text\*`.
+
+## Redirecting output
+
+### Redirect to a file
```

Comment:
Please add the heading `(Writing to an existing file)` to `Redirecting to a file`.